src/NamingStrategy/CompositeNamingStrategy.php

```diff
@@ -4,26 +4,25 @@
 
 namespace Laminas\Hydrator\NamingStrategy;
 
+use Laminas\Hydrator\NamingStrategy\NamingStrategyInterface;
+
 use function array_map;
 
 final class CompositeNamingStrategy implements NamingStrategyInterface
 {
     /** @var NamingStrategyInterface[] */
-    private $namingStrategies = [];
+    private array $namingStrategies = [];
 
-    /** @var NamingStrategyInterface */
-    private $defaultNamingStrategy;
+    private NamingStrategyInterface $defaultNamingStrategy;
 
     /**
      * @param NamingStrategyInterface[]    $strategies            indexed by the name they translate
      */
     public function __construct(array $strategies, ?NamingStrategyInterface $defaultNamingStrategy = null)
     {
         $this->namingStrategies = array_map(
-            function (NamingStrategyInterface $strategy) {
-                // this callback is here only to ensure type-safety
-                return $strategy;
-            },
+            fn(NamingStrategyInterface $strategy) => // this callback is here only to ensure type-safety
+            $strategy,
```

Comment:
I would expect `$strategy` to be on the same line - didn't even notice that it was all one closure :scream: